libs/blocks/global-navigation/global-navigation.js

```diff
@@ -250,7 +250,8 @@ export const getUniversalNavLocale = (locale) => {
 };
 
 const convertToPascalCase = (str) => str
-  ?.split('-')
+  ?.slice(1)
```

Comment:
Nit: it's not immediately obvious why this is needed, maybe replacing the starting `|` might be clearer for future readings

Reply:
Totally agree. It's more readable how you suggest. Made the change 👍